### DISCUSSION TITLE
Unicode絵文字もTwemojiに対応してほしい

### DISCUSSION THREAD

**Discussion by yarakos95** [increments/qiita-discussions#384] [要望・アイデア・不具合]
現在、Markdown emoji は Twemojiを利用して表示されていますが、Unicode 絵文字は閲覧者の環境に依存しています。

そのため、同じ絵文字を表示させようとした場合でも異なる結果を得ます。（Windows11、Google Chrome）

![image](https://user-images.githubusercontent.com/75878492/208412030-69f6c4c8-b0f8-44fa-896a-ce4601363175.png)

```markdown
☺️
:relaxed: 
```

> https://emojipedia.org/smiling-face/

また、閲覧者の Unicode バージョンによっては、Unicode 絵文字がうまく表示されない可能性があります。

**Comment by tanaka-asami415** [comment]
@Yarakashi-Kikohshi 
ご意見ありがとうございます！
Unicode をTwemoji等に変換してしまうことの方が、不自然に感じる方が多そうだと考えられるため
申し訳ありませんが、現時点では対応の予定はありません🙏
他のユーザー様からも多数ご意見としていただく場合は、改めて検討させていただきます！
今のところは、Twemojiを使用されたい時は、Twemojiをご利用いただけますと幸いです🙇
